src/inc/msquic_posix.h

```diff
@@ -505,7 +505,7 @@ QuicAddrToString(
             Address[0] = ']';
             Address++;
         }
-        sprintf(Address, ":%hu", ntohs(Addr->Ipv4.sin_port));
+        snprintf(Address, 64, ":%hu", ntohs(Addr->Ipv4.sin_port));
```

Comment:
No constant available for this?

Reply:
It's just defined right above. I don't think it's needed.